packages/tlschema/src/records/TLInstance.ts

```diff
@@ -40,6 +40,11 @@ export interface TLInstance extends BaseRecord<'instance', TLInstanceId> {
 	isFocused: boolean
 	devicePixelRatio: number
 	isCoarsePointer: boolean
+	/**
+	 * Will be null if the pointer doesn't support hovering (e.g. touch), but true or false
+	 * otherwise
+	 */
+	isHoveringCanvas: boolean | null
```

Comment:
Alternatively, we could lift the `isHoverPointer` from a CSS media query in the same way that we're lifting `isCoarsePointer`?